src/OptiHPLCHandler/applications/empower_implementation/empower_tools.py

```diff
@@ -129,3 +129,23 @@ def determine_strong_eluent(gradient_table: List[dict]) -> Optional[str]:
             list_weak_eluents.append(composition)
 
     return strong_eluent, list_weak_eluents
+
+
+def determine_decreasing_weak_eluents(gradient_table: List[dict]) -> List[str]:
+    """
+    Determine if the weak eluent is decreasing in the gradient table.
+    """
+
+    _, list_weak_eluents = determine_strong_eluent(gradient_table)
+
+    decreasing_weak_eluents = []
```

Comment:
Having plural variable names is usually a bad idea, since it is not clear what the structure is. In this case, I would prefer `decreaing_weak_eluent_list`, to make it clear that it is a list and not, say, a dict, or another type of Collection.